text/0018-publish-update.md

```diff
@@ -0,0 +1,180 @@
+
+Later on we'll want to do proactive checks for updates by submitting a list of dataset refs we'd like to grab updates for to peers and registries.
+
+#### Updating a dataset you own
+running update on a dataset in your namespace will check if a transform script is specified, If so, it'll re-execute the transform & check for changes. If changes exist it'll create a commit.
+
+Manual changes to a dataset are still run with `qri save`. Calling update on a dataset with no transform script is an error. That error should tell users to run `qri save`.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+publish and update are both steps that require an existing dataset.
+
+## Publish:
+#### CLI:
+```
+$ qri publish --help
+
+Description:
+  Qri publish makes your dataset available to others. While online peers that 
+  connect to you can only see datasets and versions that you've published. 
+  publishing a dataset always makes all previous history entries available.
+
+  By default publish also publishes your dataset to all configured registries,
+  to publish to only specific registries, use the --
+
+Examples:
+  # publish the latest version of a dataset
+  $ qri publish me/dataset
+
+  # publish a dataset (and all previous versions) at a specific version
+  $ qri publish me/dataset@/ipfs/QmFoo...
+
+  # unpublish a dataset
+  $ qri publish -d me/dataset
+
+Flags:
+  --delete -d       unpublish the specified dataset
+  --no-registry -r  don't publish to the registry
+  --registries      comma separated list of registries to publish to, overrides --no-registry
+```
+
+#### API:
+```
+# publish a reference:
+POST http://localhost:2505/publish/me/dataset
+
+# publish a dataset (and all previous versions) at a specific version
+POST http://localhost:2505/publish/me/dataset/at/ipfs/QmFoo...
+
+# unpublish a dataset
+DELETE http://localhost:2505/publish/me/dataset
+```
+
+## Update:
+#### CLI:
+```
+$ qri update --help
+
+Description:
+  Update fast-forwards your dataset to the latest known version. If the dataset
+  is owned by a peer, update will ask the peer for any new versions and download
+  them.
+
+  Calling update on a dataset you own will advance your dataset by re-running 
+  any specified transform, creating a new version of your dataset in the process.
+  If your dataset doesn't have a transform script, update does nothing.
+
+Examples:
+  # get the freshest version of a dataset from a peer
+  qri update other_person/dataset
+
+  # update your local dataset by re-running the dataset transform
+  qri update me/dataset_with_transform
+
+  # supply secrets to an update, publish on successful run
+  qri update me/dataset_with_transform -p --secrets=keyboard,cat
+
+Flags:
+  --publish, -p     publish successful updates
+  --title, -t       supply a 
```

Comment:
sentence fragment

Also, why a title but not other metadata? Commit message definitely makes sense, though.